src/main.rs

```diff
@@ -144,32 +145,48 @@ fn launch_git_with_self_as_editor() {
     run_cmd(Command::new("git").arg("commit").env("GIT_EDITOR", self_path))
 }
 
-fn collect_information_and_write_to_file(out_path: PathBuf) {
-    let maybe_emoji = select_emoji();
-
-    if maybe_emoji == None {
-        abort();
+fn git_message_is_empty(git_commit_contents: & String) -> bool {
+    for line in git_commit_contents.split("\n") {
+        if !line.starts_with('#') && line.len() > 0  {
+            return false;
+        }
     }
+    true
+}
 
-    if let Some(emoji) = maybe_emoji {
-        let mut launch_editor = false;
-        let maybe_message = collect_commit_message(emoji, &mut launch_editor);
+fn collect_information_and_write_to_file(out_path: PathBuf) {
+    let mut file = File::open(out_path.clone().into_os_string().into_string().unwrap()).unwrap();
+    let mut contents = String::new();
+    file.read_to_string(&mut contents).expect("Failed to read from file");
 
-        if maybe_message == None {
+    if git_message_is_empty(&contents) {
```

Comment:
Instead of indenting the rest of the function, can we use an early return here?

```suggestion
    if !git_message_is_empty(&mut file) {
        launch_default_editor(out_path);
        return;
    }
```